libcontainerd/remote_linux.go

```diff
@@ -283,45 +292,38 @@ func (r *remote) handleEventStream(events containerd.API_EventsClient) {
 			return
 		}
 
-		if live == false {
-			logrus.Debugf("received past containerd event: %#v", e)
-
-			// Pause/Resume events should never happens after exit one
-			switch e.Type {
-			case StateExit:
-				r.pastEvents[e.Id] = e
-			case StatePause:
-				r.pastEvents[e.Id] = e
-			case StateResume:
-				r.pastEvents[e.Id] = e
-			case stateLive:
-				live = true
-				r.updateEventTimestamp(time.Unix(int64(e.Timestamp), 0))
-			}
-		} else {
-			logrus.Debugf("received containerd event: %#v", e)
-
-			var container *container
-			var c *client
-			r.RLock()
-			for _, c = range r.clients {
-				container, err = c.getContainer(e.Id)
-				if err == nil {
-					break
-				}
-			}
-			r.RUnlock()
-			if container == nil {
-				logrus.Errorf("no state for container: %q", err)
-				continue
-			}
+		if e.Type == "live" {
```

Comment:
Can't we just have `Stored: bool`. Last event uses `true` and this uses `false`.